### PR TITLE
Don't close streams

### PR DESCRIPTION
**Pull request by linuskendall**
Removes the work arounds for the bidi streams not working as expected due to our loadbalancer bug.

### CODE REVIEW

File: golang/cmd/grpc-client/main.go [hunk 1]
```diff
@@ -232,9 +232,18 @@ func grpc_subscribe(conn *grpc.ClientConn) {
 	if err != nil {
 		log.Fatalf("%v", err)
 	}
-	stream.CloseSend()
 
+  i := 0
```
Comment (fanatid) [inline golang/cmd/grpc-client/main.go]: Do we really need this counter?

Comment (linuskendall) [reply golang/cmd/grpc-client/main.go]: No, before merging we should come up with some nice way to demonstrate re-subs in the clients?

Comment (fanatid) [reply golang/cmd/grpc-client/main.go]: maybe some way as in rust example? left counter but comment line where we send new subscribe config?

Comment (linuskendall) [reply golang/cmd/grpc-client/main.go]: I've added a new resub param. Good to keep this option for testing and example illustration.

File: rust/client/client.rs [hunk 2]
```diff
@@ -295,6 +304,11 @@ async fn main() -> Result<(), Error> {
         blocks_meta.insert("client".to_owned(), SubscribeRequestFilterBlocksMeta {});
     }
 
+    let mut resub: u16 = 0;
```
Comment (fanatid) [inline rust/client/client.rs:307]:
```
let resub = args.resub.unwrap_or(0)
```
:slightly_smiling_face:

**Pull request merged**